### ISSUE TITLE
NMT-Keras dockerfile build failing

### ISSUE THREAD

**Issue by jiwidi** [midobal/dockerfiles#1]
Hi!

First of all, thank you for the repo. I want to use your dockerfile to train models locally for the TA course but I can't build your image. 

I'm running 
```
docker build -t nmt-keras .
``` 
With your dockerfile and fails on:

```
/bin/sh: 1: pip: not found
The command '/bin/sh -c wget https://repo.continuum.io/miniconda/Miniconda3-py37_4.8.3-Linux-x86_64.sh -O /tmp/miniconda.sh && bash /tmp/miniconda.sh -b -p /opt/miniconda && export PATH="/opt/miniconda/bin:$PATH" && conda config --set always_yes yes --set changeps1 no && conda update -q conda && conda init && pip install -e . && conda install tensorflow-gpu=1.15' returned a non-zero code: 127
```

Any idea of why?

**Comment by midobal** [comment]
Hi!

Actually, I detected this error recently on the installation script of the [TA version](https://github.com/PRHLT/nmt-keras_practicas-TA) and I didn't realize that it also affected this Dockerfile. Thanks for letting me know!

The problem is that, for some reason, "conda update" now uninstalls pip. Thinking carefully, updating conda in a docker may not be the best of ideas. I just made a new commit that should have corrected this. Let me know otherwise.

P.S. For reasons that I will explain at tomorrow's class, don't use this Dockerfile for the TA course. A new Dockerfile will be ready at the TA's [repo](https://github.com/PRHLT/nmt-keras_practicas-TA) at some point during this evening.
